src/providers/arcgisrest/qgsamsprovider.cpp

```diff
@@ -84,7 +84,7 @@ void QgsAmsLegendFetcher::start()
     const QString referer = dataSource.param( QStringLiteral( "referer" ) );
     QgsStringMap headers;
     if ( !referer.isEmpty() )
-      headers[ QStringLiteral( "Referer" )] = referer;
+      headers[ QStringLiteral( "referer" )] = referer;
```

Comment:
what's the rationale behind this change?

Reply:
Everywhere else, "referer" is used without the uppercase. It is more a homogenization than a bug correction.